### DISCUSSION TITLE
two DynUNet?

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#516] [Q&A] [answered]
What's the difference between **monai.networks.nets.dynunet_v1.DynUNet** and **monai.networks.nets.DynUNet**? And I can't find the source code for **monai.networks.nets.dynunet_v1**, where is it?

**Comment by SachidanandAlle** [comment, marked as answer]
monai.networks.nets.dynunet_v1 is deprecated.. please use monai.networks.nets.dynunet
if you want to see.. then refer 0.7 version code for MONAI
https://github.com/Project-MONAI/MONAI/blob/0.7.0/monai/networks/nets/dynunet_v1.py

**Reply by hw-ju** [reply]
@SachidanandAlle Thanks!! 
I see **monai.networks.nets.dynunet_v1** in 
https://github.com/Project-MONAI/MONAILabel/blob/fb2a4b976bed06bf1ed3bc6662602a9aff95d3b4/sample-apps/deepedit/main.py 
and in https://github.com/Project-MONAI/MONAILabel/tree/fb2a4b976bed06bf1ed3bc6662602a9aff95d3b4/sample-apps/deepedit
